.coveragerc

```diff
@@ -0,0 +1,19 @@
+[report]
+exclude_lines =
+    # Skip any pass lines such as may be used for @abstractmethod
```

Comment:
this got me to 100% test coverage (I am running this command btw: `pytest --cov --cov-report=term-missing`)

Reply:
https://github.com/pytest-dev/pytest-cov/issues/428#issuecomment-821063750